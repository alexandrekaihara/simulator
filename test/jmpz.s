LOAD ZERO
JMPZ ISZERO
OUTPUT MINUSONE
STOP
ISZERO: OUTPUT ONE
STOP
ZERO: CONST 0 
ONE: CONST 1
MINUSONE: CONST -1