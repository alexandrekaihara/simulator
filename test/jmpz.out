10 10 8 7 13 12 14 13 11 14 0 1 -1